libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -991,8 +1045,8 @@ bool AP_Mission::mission_cmd_to_mavlink(const AP_Mission::Mission_Command& cmd,
         packet.param2 = cmd.content.digicam_configure.shutter_speed;
         packet.param3 = cmd.content.digicam_configure.aperture;
         packet.param4 = cmd.content.digicam_configure.ISO;
-        packet.x = cmd.content.digicam_configure.exposure_type;
-        packet.y = cmd.content.digicam_configure.cmd_id;
+        packet.x = cmd.content.digicam_configure.exposure_type * 1.0e7f;
```

Comment:
Not sure if this is correct or not. It's listed as identity/enum's, which seems to me implies it was an integer value to begin with, so multiplying it by 1e7 seems off, but its an under defined area of the protocol. As I read it thins that are enums/identities and aren't positions in space shouldn't be seeing a 1e7 multiplier, but I'm interested what other people think?